root_locals.tf

```diff
@@ -120,8 +120,8 @@ locals {
   internal_bucket_key_enabled    = local.environment == "intg"
 
   // s3 upload bucket encryption
-  upload_dirty_s3_encryption_key_arn = ""
-  upload_dirty_bucket_key_enabled    = false
+  upload_dirty_s3_encryption_key_arn = local.environment == "intg" ? module.s3_upload_kms_key.kms_key_arn : ""
+  upload_dirty_bucket_key_enabled    = true
```

Comment:
Think this should be behind a ternary as well? Though should this be set to true regardless?

Reply:
Thanks for the spot!